### PR TITLE
MINOR: Rename OmvDataSource to VectorTileDataSource.

### PR DESCRIPTION
**Pull request by Astrak**
Is the world ready for this?

### CODE REVIEW

File: yarn-error.log [hunk 1]
```diff
@@ -0,0 +1,9071 @@
+Arguments:
```
Comment (zbigg) [inline yarn-error.log]: This file is not needed, remove it.

File: @here/harp-omv-datasource/lib/OmvRestClient.ts [hunk 2]
```diff
@@ -171,295 +172,16 @@ export const AuthenticationTypeAccessToken: AuthenticationMethodInfo = {
...
-
-/**
- * REST client supporting getting protobuf OMV Tile from REST-based servers.
- */
-export class OmvRestClient implements DataProvider {
-    private readonly downloadManager: TransferManager;
-    private readonly urlParams: { [key: string]: string };
-
+let warningShown = false;
+export class OmvRestClient extends VectorTileRestClient {
```
Comment (nzjony) [inline @here/harp-omv-datasource/lib/OmvRestClient.ts]: add @depracated to the class comment?

File: @here/harp-vectortile-datasource/lib/VectorTileRestClient.ts [hunk 3]
```diff
@@ -0,0 +1,465 @@
...
+     *
+     * Sample URL:
+     * `https://xyz.api.here.com/tiles/osmbase/256/all/16/19293/24641.json?access_token=your-xyz-api-key`
+     *
+     * Default authentication method used: [[AuthenticationTypeAccessToken]].
+     */
+    XYZJson,
+
+    /**
+     * Use the REST API format of XYZ Vector Tile API in OMV format.
```
Comment (nzjony) [inline @here/harp-vectortile-datasource/lib/VectorTileRestClient.ts]: Would it be worth adding a new format and deprecating XYZOMV

File: @here/harp-examples/src/datasource_geojson_styling_game.ts [hunk 4]
```diff
@@ -14,20 +14,20 @@ import {
...
 import { accessToken, copyrightInfo } from "../config";
 import * as geojson from "../resources/italy.json";
 
 /**
- * In this example showcases how to use the styling of a [[OmvDataSource]] that using a
+ * In this example showcases how to use the styling of a [[VectorTileDataSource]] that using a
  * [[GeoJsonDataProvider]] with a dynamic [[StyleSet]] to generate a quiz game. First, we generate a
  * [[MapView]], without [[MapControls]] this time, and then we also attach an additional base map
- * from an [[OmvDataSource]].
+ * from an [[VectorTileDataSource]].
```
Comment (CraigBerlin) [inline @here/harp-examples/src/datasource_geojson_styling_game.ts]: a

File: @here/harp-examples/src/styling_data-driven.ts [hunk 5]
```diff
@@ -93,7 +93,7 @@ export namespace DataDrivenThemeExample {
     function main() {
         const map = initializeMapView("mapCanvas");
 
-        const omvDataSource = new OmvDataSource({
+        const baseMapData = new VectorTileDataSource({
```
Comment (CraigBerlin) [inline @here/harp-examples/src/styling_data-driven.ts]: For consistency in examples should use either "baseMapData" or "mapData" everywhere.

File: yarn-error.log (hunk 1 above)
Comment (Astrak) [reply yarn-error.log]: I thought I could grab a berth on the contributors ranking with this +9000 file, too bad I got called out!

File: @here/harp-vectortile-datasource/lib/VectorTileRestClient.ts (hunk 3 above)
Comment (Astrak) [reply @here/harp-vectortile-datasource/lib/VectorTileRestClient.ts]: Yes that's the topic of the discussion to have tomorrow

**Comment by Astrak** [comment]
Smaller, better and more targetted PRs will be planned after all the solutioning is ready.

**Pull request closed without merging**